guides/user/kubernetes-v2/rollout-strategies/index.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: single
+title:  "Rollout Strategies"
+sidebar:
+  nav: guides
+---
+
+{% include alpha version="1.14" %}
+
+{% include toc %}
+
+This guide describes how to take advantage of the
+[Kubernetes V2](/setup/install/providers/kubernetes-v2) provider's first-class support
+for common rollout strategies, including dark, highlander, and red/black rollouts.
```

Comment:
Of course, Highlander's not there either. :/

Reply:
Hmm, if highlander isn't there, maybe it's meant to just illustrate a few common patterns rather than be an exhaustive list of all patterns implemented by all cloud providers? Not sure how that image was created but looks like the commit was made by @skim1420, so will defer to him if he would like us to update it!